APIManager.js

```diff
@@ -0,0 +1,89 @@
+class APIManager {
+  constructor() {
+    this.data = {}
+  }
+
+  callApis(apiManager) {
+    $.ajax({
+      url: "https://randomuser.me/api/?results=7",
+      success: function (response) {
+        let userData = response.results[0]
+        userData = {
+          imageURL: userData.picture.medium,
+          firstName: userData.name.first,
+          lastName: userData.name.last,
+          cityState: userData.location.city + "," + userData.location.state,
+        }
+        let friendsData = response.results.slice(1)
+        friendsData = friendsData.map((m) => {
+          return { firstName: m.name.first, lastName: m.name.last }
+        })
+        apiManager.data["_userInfo"] = userData
+        apiManager.data["_friends"] = friendsData
```

Comment:
There is no reason to call the property with an underscore `_userInfo ` can be `userInfo `